rocketchat-core/src/main/java/com/rocketchat/core/ChatRoom.java

```diff
@@ -155,10 +172,11 @@ public void setFavourite(Boolean isFavoutite, SimpleCallback callback) {
     //Subscription methods
 
     public void subscribeRoomMessageEvent(SubscribeListener subscribeListener,
-                                          MessageCallback.SubscriptionCallback callback) {
+                                          MessageCallback.SubscriptionListener callback) {
         if (roomSubId == null) {
             roomSubId = client.subscribeRoomMessageEvent(room.roomId(),
                     true, subscribeListener, callback);
+            deleteSubId = client.subscribeRoomDeleteEvent(room.roomId(), true, null);
```

Comment:
why null on the subscriber? the user is never notified of the unsubscription?

Reply:
It is unnecessary separate subscription created for receiving deleted messages. It has to be subscribed whenever user subscribes for receiving messages.